README.md

```diff
@@ -285,9 +285,9 @@ cargo test --verbose # auth tests included
 #### Topology-specific tests
 Certain tests will only be run against certain topologies. To ensure that the entire test suite is run, make sure to run the tests separately against standalone, replicated, and sharded deployments.
 ```bash
-export MONGODB_URI="mongodb://my-standalone-host:20717" # mongod running on 27017
+export MONGODB_URI="mongodb://my-standalone-host:27017" # mongod running on 27017
 cargo test --verbose
-export MONGODB_URI="mongodb://localhost:27018,localhost:27019,localhost:27020/?replSet=repl"
+export MONGODB_URI="mongodb://localhost:27018,localhost:27019,localhost:27020/?replicaSet=repl" # replicaset running on 27018 with name repl
```

Comment:
The replica set here consists of three mongod nodes, one running on port 27018, one on port 27019, and one on port 27020, so it isn't completely accurate to say the replica set is running on 27018. Instead, we could say `replicaset running on ports 27018, 27019, and 27020 with set name "repl"`.

Reply:
works for me!